src/jswrap_object.c

```diff
@@ -1157,10 +1157,13 @@ This executes the function with the supplied 'this' argument and parameters
   "name" : "apply",
   "generate" : "jswrap_function_apply_or_call",
   "params" : [
-    ["this","JsVar","The value to use as the 'this' argument when executing the function"],
-    ["args","JsVar","Optional Array of Arguments"]
+    ["thisArg","JsVar","The value to use as the 'this' argument when executing the function"],
+    ["args","JsVarArray","Optional Array of Arguments"]
```

Comment:
Have you tested this? I'm pretty sure this breaks the build.

These headers are not just for Typescript and documentation - they tell the interpreter what to expect. And JsVarArray isn't a JsVar that's an array, it's more like a spread operator

Reply:
Ah yes - I had totally missed that, thank you